packages/cubejs-teradata-jdbc-driver/bin/installer.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+var __importDefault = (this && this.__importDefault) || function (mod) {
+    return (mod && mod.__esModule) ? mod : { "default": mod };
+};
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.downloadJDBCDriver = void 0;
+const path_1 = __importDefault(require("path"));
+const inquirer_1 = __importDefault(require("inquirer"));
+const shared_1 = require("@cubejs-backend/shared");
+function acceptedByEnv() {
+    const acceptStatus = shared_1.getEnv('teradataAcceptPolicy');
+    if (acceptStatus) {
+        console.log('You accepted Terms & Conditions for JDBC driver from Teradata by CUBEJS_DB_TERADATA_ACCEPT_POLICY');
+    }
+    if (acceptStatus === false) {
+        console.log('You declined Terms & Conditions for JDBC driver from Teradata by CUBEJS_DB_TERADATA_ACCEPT_POLICY');
+        console.log('Installation will be skipped');
+    }
+    return acceptStatus;
+}
+async function cliAcceptVerify() {
+    console.log('Teradata driver is using JDBC driver from Teradata');
+    console.log('By downloading the driver, you agree to the Terms & Conditions');
+    console.log('link to insert');
+    console.log('More info: say no more');
+    if (process.stdout.isTTY) {
+        const { licenseAccepted } = await inquirer_1.default.prompt([{
+                type: 'confirm',
+                name: 'licenseAccepted',
+                message: 'You read & agree to the Terms & Conditions',
+            }]);
+        return licenseAccepted;
+    }
+    shared_1.displayCLIWarning('Your stdout is not interactive, you can accept it via CUBEJS_DB_DATABRICKS_ACCEPT_POLICY=true');
+    return false;
+}
+async function downloadJDBCDriver(isCli = false) {
+    let driverAccepted = acceptedByEnv();
+    if (driverAccepted === undefined && isCli) {
+        driverAccepted = await cliAcceptVerify();
+    }
+    if (driverAccepted) {
+        console.log('Downloading terajdbc4');
+        await shared_1.downloadAndExtractFile('https://drive.google.com/uc?export=download&id=11HlWnmzCJ7S5zFRYA_FWHFdz0gfs5Isj', {
```

Comment:
1. We should probably externalize this constant elsewhere (hint: a new file on the repo), so the next time we update the link we don't have to update the code, but only the resource file that is pointing to the updated URL.

2. How will it work in the bank exactly?
    Can we point to a local file on disk? will it work? Have you tested it?

Reply:
This problem is not an issue now. We have the .jar file in the server and the .jar file is also already in the container